package.json

```diff
@@ -85,7 +85,7 @@
         "@no-repeat/sassdoc-parser": "^0.0.11",
         "@octokit/rest": "^15.15.1",
         "autoprefixer": "^7.1.4",
-        "axe-core": "3.1.2",
+        "axe-core": "^3.1.2",
```

Comment:
Change this to 3.2.0 and then run npm install and test

Reply:
This needs to be changed to 3.2 because we need developers to install the latest